xml/System.Resources.Extensions/PreserializedResourceWriter.xml

```diff
@@ -345,8 +404,9 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Write out all resources to the output stream in the system default format. If an exception occurs during object serialization or during IO, the .resources file is closed and deleted, since it is most likely invalid.</summary>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>  
          <format type="text/markdown"><![CDATA[  

If an exception occurs during object serialization or during IO, the .resources file is closed and deleted, since it is most likely invalid.

         ]]></format>
         </remarks>
```